src/App.js

```diff
@@ -3,33 +3,69 @@ import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 import InputWithLabel from "./InputWithLabel";
 
-function useSemiPersistentState() {
-  const savedTodoList = JSON.parse(localStorage.getItem("savedTodoList")) || [];
-  const [todoList, setTodoList] = useState(savedTodoList);
+// function useSemiPersistentState() {
+//   const savedTodoList = JSON.parse(localStorage.getItem("savedTodoList")) || [];
+//   const [todoList, setTodoList] = useState(savedTodoList);
+
+//   useEffect(() => {
+//     localStorage.setItem ("savedTodoList", JSON.stringify(todoList));
+//   }, [todoList]);
+//     return [todoList,setTodoList];
+// }
 
-  useEffect(() => {
-    localStorage.setItem ("savedTodoList", JSON.stringify(todoList));
-  }, [todoList]);
-    return [todoList,setTodoList];
-}
 
 function App() {
-  const [todoList, setTodoList] = useSemiPersistentState();
+   const [todoList, setTodoList] = useState([]);
+   const [isLoading, setIsLoading] = useState(true);
+   const [workInProgress, serWorkInProgress] = useState(true);
```

Comment:
Nitpick:
This state is always true since `serWorkInProgress` is never used. I’m not sure what the intention of this state is, as such, I’d recommend removing it.

Reply:
Yes, I was trying to add stretching skills , but yes, I'll remove it . Thank you so much Matt